src/main/kotlin/no/nav/tiltaksarrangor/service/MetricsService.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.tiltaksarrangor.service
+
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Tags
+import org.springframework.stereotype.Service
+
+private const val innlogging_metric = "tiltaksarrangorbff_innlogging"
+
+@Service
+class MetricsService(
+	registry: MeterRegistry
+) {
+	private val reg = registry
+
+	fun incInnloggetAnsatt(count: Int = 1, roller: List<String>) {
```

Comment:
Jeg vet ikke helt om jeg skjønner helt her. Hvor øker counten? En gauge viser vel bare nummeret du gir den?

Reply:
Den gjør kanskje det..? Hvordan får jeg tags/labels på en counter med dette oppsettet? Jeg har vært vant til å gjøre sånn som dette: 

val MIN_COUNTER: Counter =
    Counter.Builder()
        .namespace("tiltaksarrangorbff")
        .name("innlogging")
        .labelNames("rolle")
        .register()

MIN_COUNTER.labels("KOORDINATOR").inc()

Men det er visst ikke sånn man gjør ting med spring :/

Reply:
Metrikker er jeg skikkelig dårlig på, jeg ser i amt-tiltak at det blir gjort en databasespørring for å finne nåværende antall innlogginger.

Reply:
ja, og det blir det ikke i denne appen 😅 Jeg vil ha en counter som trigger når kode kjøres. Uten spring er det kjempelett, så jeg må bare finne ut hvordan jeg gjør det med spring. Skal vi ha tall fra databasen får vi sette opp et dataprodukt for det :)

Reply:
Ser ut som at det jeg har brukt hittil er prometheus, mens det jeg får med spring boot er micrometer. Fant en guide nå. Det blir litt mindre fint enn med prometheus, men skal nok få det til :)

Reply:
tror det kanskje ble bedre nå?

Reply:
Hmm, vi får ikke telt unike brukere da 🤔

Reply:
Jeg tror ikke egentlig prometheus er riktig verktøy for å telle brukere. Kanskje vi heller bør bruke amplitude for dette?

Reply:
Kanskje det går an ja, vet ikke om det er noen nye muligheter med de frontend greiene fra nais? Fint å ha ting i grafana.